.env

```diff
@@ -2,11 +2,13 @@
 # SPDX-License-Identifier: GPL-2.0-only
 # Copyright (c) 2022 Intel Corporation.
 
-lkvs_root=$(cd "$(dirname "$0")" && pwd)
+lkvs_root=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
 
 if [[ -z "$path_exported" ]]; then
-  NEW_PATH="${PATH}":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
+  NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
   NEW_PATH=${NEW_PATH%:}
   export PATH="$NEW_PATH"
   path_exported=1
 fi
+
+source common.sh
```

Comment:
Should move "source common.sh"  into the if judgement,  like after ' export PATH="$NEW_PATH" '  line.

Reply:
I tried  cpu_bat.sh  call  cpu_common.sh  without issue.

And thanks for comments,  shall we need to add  "source common.sh" in .env.

It's ok for me.

Thanks!

Reply:
> Should move "source common.sh" into the if judgement, like after ' export PATH="$NEW_PATH" ' line.

I prefer to put `source common.sh` out of the *if* block, because after changing code in *common.sh*, we can easily import the changes into current environment (terminal) by `source .env`. This is important when debugging/developping code in *common.sh*. For example, if I want reimplement function `test_print_trc`, I will: 1. change the code. 2. run `source .env` in termial. 3.  run `test_print_trc "some log"`, 4. repeat 1, 2, 3.